### PR TITLE
Update link to Zephyr led docs

### PR DESCRIPTION
**Pull request by manthonygfp**
The documentation has a broken link. This change updates the link to a relevant page.

### CODE REVIEW

File: docs/docs/config/backlight.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Applies to: [`/chosen` node](https://docs.zephyrproject.org/latest/build/dts/int
 
 See the Zephyr devicetree bindings for LED drivers:
 
-- [gpio-leds](https://docs.zephyrproject.org/latest/build/dts/api/bindings/gpio/gpio-leds.html)
+- [gpio-leds](https://docs.zephyrproject.org/latest/build/dts/api/bindings/led/gpio-leds.html)
```
Comment (petejohanson) [inline docs/docs/config/backlight.md:37]: While we are at it, can we instead link to the 3.0 release version docs, so we're not looking to a moving target with the `latest` piece?

**Review by petejohanson — CHANGES_REQUESTED** [review]
Thanks for the fix! Just one suggestion.

File: docs/docs/config/backlight.md (hunk 1 above)
Comment (manthonygfp) [reply docs/docs/config/backlight.md]: I wasn't sure if you wanted to use the 3.0.0 or use whatever the latest is for 3.0.x. If this isn't good, I'll have to figure out a more clever way to always be selecting 3.0.x.

Comment (petejohanson) [reply docs/docs/config/backlight.md]: Fixed at 3.0.0 is fine, it should be API stable for 3.0.x.

**Review by petejohanson — APPROVED** [review]
Thanks!

**Pull request merged**